docs/SPEC.md

```diff
@@ -343,24 +314,22 @@ A regular expression literal represents a regular expression pattern, enclosed i
 Within the forward slashes, any unicode character may appear except for an unescaped forward slash.
 The `\x` hex byte value representation from string literals may also be present.
 
-Regular expression literals support only the following escape sequences:
+In addition standard escape sequences regular expression literals also support the following escape sequences:
 
     \/   U+002f forward slash
-    \\   U+005c backslash
-
 
     regexp_lit         = "/" regexp_char { regexp_char } "/" .
     regexp_char        = unicode_char | byte_value | regexp_escape_char .
-    regexp_escape_char = `\` (`/` | `\`)
+    regexp_escape_char = `\/`
+
 
 Examples:
 
     /.*/
     /http:\/\/localhost:8086/
     /^\xe6\x97\xa5\xe6\x9c\xac\xe8\xaa\x9e(ZZ)?$/
     /^日本語(ZZ)?$/ // the above two lines are equivalent
-    /\\xZZ/ // this becomes the literal pattern "\xZZ"
-    /a\/b\\c\d/ // escape sequences and character class shortcuts are supported
```

Comment:
There is no need to escape backslash in regex and it was never implemented, The SPEC has been updated to reflect that.